extensions/common/aws/aws-s3-core/src/main/java/org/eclipse/edc/aws/s3/S3CoreExtension.java

```diff
@@ -45,6 +45,8 @@ public class S3CoreExtension implements ServiceExtension {
     private static final String AWS_ENDPOINT_OVERRIDE = "edc.aws.endpoint.override";
     @Setting(value = "The size of the thread pool used for the async clients")
     private static final String AWS_ASYNC_CLIENT_THREAD_POOL_SIZE = "edc.aws.client.async.thread-pool-size";
+    @Setting(value = "If true, path style addressing is used instead of virtual-hosted style")
```

Comment:
please add the "type" and the "default" attributes

Reply:
added type and defaultValue.